File: CS/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraReports.UI;

namespace LimitNumberOfRecordsPerPage {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e) {

            XtraReport1 detailreport = new XtraReport1();
            ReportPrintTool printTool = new ReportPrintTool(detailreport);
            printTool.ShowPreview();
        }
    }
}

File: VB/Form1.vb
Imports System
Imports System.Windows.Forms
Imports DevExpress.XtraReports.UI

Namespace LimitNumberOfRecordsPerPage
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
            Dim detailreport As New XtraReport1()
            Dim printTool As New ReportPrintTool(detailreport)
            printTool.ShowPreview()
        End Sub
    End Class
End Namespace